sg.tf

```diff
@@ -1,6 +1,6 @@
 //security group for load balancer
-resource "aws_security_group" "sg-lb" {
-  vpc_id = aws_vpc.test-vpc.id
+resource "aws_security_group" "sg_lb" {
```

Comment:
Rename to `aws_security_group.app_lb`

Reply:
Renamed load balancer security group from sg_lb to app_lb